app/services/report_pdf_builder.rb

```diff
@@ -6,15 +6,18 @@ def initialize(report)
   end
 
   def run
+    changes_grouped_by_member = @report.members.map(&:reported_changes).flatten
```

Comment:
Sharon wants all the changes to be grouped by member, before they were just in user entered order.